### PR TITLE
Fix Deprecations

### PR DESCRIPTION
**Pull request by cdmwebs**
cc @rockwood

### CODE REVIEW

**Comment by rockwood** [comment]
Thanks @cdmwebs!

The deprecations are gone, but it looks like something got messed up with the map tiles. It appears to be repeating the same tile. Screenshot attached.

image

**Comment by cdmwebs** [comment]
@rockwood turns out you need Leaflet's CSS, too.

File: app/components/bd-map.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
-import Leaflet from 'bus-detective/utils/leaflet';
```
Comment (rockwood) [inline app/components/bd-map.js]: I created this wrapper module so we're aren't referencing the window variable. Is there any reason we can't use that?

**Comment by cdmwebs** [comment]
I don't see why not @rockwood. Pushed. Also, this should fix #5, too.

**Pull request closed without merging**